bridges/testing/environments/rococo-westend/spawn.sh

```diff
@@ -43,6 +43,15 @@ if [[ $init -eq 1 ]]; then
   westend_init_log=$logs_dir/westend-init.log
   echo -e "Setting up the westend side of the bridge. Logs available at: $westend_init_log\n"
 
+  $helper_script init-rococo-local >> $rococo_init_log 2>&1 &
+  rococo_init_pid=$!
+  $helper_script init-westend-local >> $westend_init_log 2>&1 &
+  westend_init_pid=$!
+  wait -n $rococo_init_pid $westend_init_pid
```

Comment:
Looks to me like the intent is to wait for all to finish before proceeding. The `-n` parameter will cause to proceed as soon as one of these finishes
```suggestion
  wait $rococo_init_pid $westend_init_pid
```

Reply:
@karolk91 good point, I don't know, but I have a feeling that we don't need to really wait for both pids, basically here we just fire some scripts/extrinsics, and later we have really waiting point `run_zndsl ${BASH_SOURCE%/*}/rococo-init.zndsl` where we wait for the data.

Maybe this `-n` is kind of intentional here. @serban300 Don't you remember? I think your or Slava were adding this.